src/pyasm/prod/service/api_xmlrpc.py

```diff
@@ -1007,6 +1014,44 @@ def log(self, ticket, level, message, category="default"):
 
 
 
+    def browser_res(self, meth, result):
+
+        web = WebContainer.get_web()
+        if (web and web.get_app_name() == "Browser" \
+                and meth.__name__ not in ['get_widget'] \
+                and self.get_language() == 'javascript'):
+            result = jsondumps(result)
+
+        return result
+
+
+
+    #@trace_decorator
+    def redirect_to_server(self, ticket, meth, args):
+
+        project_code = Config.get_value("master", "project_code")
+        url = Config.get_value("master", "url") 
+        rest_url = "http://" + url + "/" + project_code + "/REST/"
+        cmd = args[0]
```

Comment:
Also, not necessary for initial implementation, but what if there is a site? Not just project code.